src/main/java/org/broadinstitute/consent/http/db/SamDAO.java

```diff
@@ -97,17 +97,13 @@ public UserStatus postRegistrationInfo(AuthUser authUser) throws Exception {
     HttpResponse response = executeRequest(request);
     String body = response.parseAsString();
     if (!response.isSuccessStatusCode()) {
-      if (HttpStatusCodes.STATUS_CODE_CONFLICT == response.getStatusCode()) {
```

Comment:
removed this as a separate case because it was being caught and generalized in the OAuthAuthenticator anyways